aten/src/ATen/native/cpu/Loops.h

```diff
@@ -219,7 +219,7 @@ void cpu_kernel_vec(TensorIterator& iter, func_t&& op, vec_func_t&& vop) {
         }
       });
     }
-  });
+  }, 1024);
```

Comment:
Please declare as constant explaining selection approach.

Reply:
I have removed grain size for loop in this PR